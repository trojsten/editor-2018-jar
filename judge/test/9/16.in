VECTOR_INT:
lode
96
61
58
52
9
36
69
24
42
17
47
12
64
49
3
10
50
53
83
67
90
34
39
38
86
31
40
96
30
95
77
2
65
37
92
41
8
78
48
76
79
11
75
93
32
35
85
22
23
25
1
66
13
88
15
89
55
60
26
27
80
84
44
59
63
71
72
70
5
43
56
46
73
62
33
6
20
57
28
19
81
45
51
91
4
87
18
21
29
68
94
7
74
14
82
16
54
INT:
nepouzijestentointnadarmo
0
