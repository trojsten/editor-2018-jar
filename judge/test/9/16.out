{"lode": [1, 25, 23, 22, 85, 35, 32, 93, 75, 11, 79, 76, 48, 78, 8, 41, 92, 37, 65, 2, 77, 95, 30, 96, 40, 31, 86, 38, 39, 34, 90, 67, 83, 53, 50, 10, 3, 49, 64, 12, 47, 17, 42, 24, 69, 36, 9, 52, 58, 61, 54, 16, 82, 14, 74, 7, 94, 68, 29, 21, 18, 87, 4, 91, 51, 45, 81, 19, 28, 57, 20, 6, 33, 62, 73, 46, 56, 43, 5, 70, 72, 71, 63, 59, 44, 84, 80, 27, 26, 60, 55, 89, 15, 88, 13, 66]}